alpha/apps/kaltura/lib/myPlaylistUtils.class.php

```diff
@@ -955,7 +955,16 @@ public static function isEntryInPlaylist($entryId, $playlistId, $partnerId)
 			// assume static playlist
 			$static_playlist_str = $playlistEntry->getDataContent();
 			$static_playlist = explode ( "," , $static_playlist_str );
-			if(in_array($entryId, $static_playlist)) return true;
+			if(in_array($entryId, $static_playlist))
+				return true;
+
+			//check if entryId is redirectEntryId of an entry in the playlist
+			$playlistEntries = entryPeer::retrieveByPKs($static_playlist);
```

Comment:
Since this method provides the info of isEntryInPlaylist, then it is wrong to put this code here.
Please add another fucntion - isEntryRereferedByPlaylist(entryId,PlaylistId)